scripts/upgrade

```diff
@@ -122,23 +111,23 @@ ynh_system_user_create --username=$app
 # CREATE DIRECTORIES
 #=================================================
 
-mkdir -p /home/yunohost.$app/{Podcasts,Playlists}
+mkdir -p /home/yunohost.app/$app/{Podcasts,Playlists}
```

Comment:
With this new folder, what happen to people who have their files in the old one?

You should move the content of the old one to the new one?
Maybe also removing the old one but in the case, what happen what the airsonic behavior?

Reply:
+1, I think we can add : 

```bash
# Rename legacy folder to proper location
if [[ ! -e /home/yunohost.app/$app ]] && [[ -e /home/yunohost.$app ]]
then
    mkdir -p /home/yunohost.app/
    mv /home/yunohost.$app /home/yunohost.app/$app
fi
```